doc/install/python.rst

```diff
@@ -66,77 +71,74 @@ Every time you make changes to the code, you'll have to repeat the second step.
 Advanced options
 ^^^^^^^^^^^^^^^^
 
-By default Arbor is installed with multi-threading enabled.
-To enable more advanced forms of parallelism, the following optional flags can
-be used to configure the installation:
+By default Arbor is installed with multi-threading enabled. To enable more advanced forms of parallelism,
+Arbor comes with a few compilation options. These can be used on both local (``pip3 install ./arbor``) and
+remote (``pip3 install arbor``) copies of Arbor. Below we assume you are working off a local copy.
+
+The following optional flags can be used to configure the installation:
 
 * ``--mpi``: Enable MPI support (requires MPI library).
 * ``--gpu``: Enable GPU support for NVIDIA GPUs with nvcc using ``cuda``, or with clang using ``cuda-clang`` (both require cudaruntime).
   Enable GPU support for AMD GPUs with hipcc using ``hip``. By default set to ``none``, which disables gpu support.
-* ``--vec``: Enable vectorization. This might require choosing an appropriate architecture using ``--arch``. Note that on x86-64 platforms compilation will fail if you enable vectorization, but the CPU or ``--arch`` does not support any form of AVX.
-* ``--arch``: CPU micro-architecture to target. By default this is set to ``native``.
-
-If calling ``setup.py`` the flags must come after ``install`` on the command line,
-and if being passed to pip they must be passed via ``--install-option``. The examples
-below demonstrate this for both pip and ``setup.py``.
+* ``--vec``: Enable vectorization. This requires choosing an appropriate architecture using ``--arch``.
+  Note that on x86-64 platforms compilation will fail if you enable vectorization, but the CPU or ``--arch`` does not support any form of AVX.
```

Comment:
Instead, redirect the readers to existing documentation for the architecture flag:

https://docs.arbor-sim.org/en/latest/install/build_install.html#architecture